xmtp_db/src/encrypted_store/group/dms.rs

```diff
@@ -0,0 +1,124 @@
+use super::*;
+
+impl DbConnection {
```

Comment:
I separated some of the larger storage items into multiple files. For instance, group into `group/convert`, `group/dms`, `group/version` or `group_message/tests`. DM queries + tests would be in `group/dms` for instance, while all conversions of database items between db/protobuf are in `convert.rs`